mockall_derive/src/lib.rs

```diff
@@ -823,6 +830,8 @@ impl<'a> AttrFormatter<'a> {
                     self.doc
                 } else if *i.as_ref().unwrap() == "async_trait" {
                     self.async_trait
+                } else if *i.as_ref().unwrap() == "make" {
```

Comment:
It's unfortunate that the most significant word in this attribute name isn't the last one "make", but the first "trait_variant".  That's unlike every other attribute that we process.  And we may have to alter our processing in the future, if another crate defines an attribute called "make".  But for now we can leave it.